opentelekomcloud/resource_opentelekomcloud_csbs_backup_policy_v1.go

```diff
@@ -221,28 +222,35 @@ func resourceCSBSBackupPolicyRead(d *schema.ResourceData, meta interface{}) erro
 			return nil
 		}
 
-		return fmt.Errorf("Error retrieving backup policy: %s", err)
+		return fmt.Errorf("error retrieving backup policy: %s", err)
 	}
 
 	if err := d.Set("resource", flattenCSBSPolicyResources(*backupPolicy)); err != nil {
-		return err
+		return fmt.Errorf("[DEBUG] Error saving resource to state for OpenTelekomCloud CSBS backup policy (%s): %s", d.Id(), err)
```

Comment:
Too much overlapping between error messages, you can move the pattern to const section and pass the used field to that pattern